From 1364827693f34a7343212844f6112f514870da14 Mon Sep 17 00:00:00 2001
From: Evan Huus <eapache@gmail.com>
Date: Fri, 15 Mar 2013 22:09:51 +0000
Subject: [PATCH 6/7] Fix
 https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=8481

Trivially wrong format string being passed to val_to_str().

svn path=/trunk/; revision=48332

Conflicts:
	epan/dissectors/packet-mpeg-dsmcc.c
---
 epan/dissectors/packet-mpeg-dsmcc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/epan/dissectors/packet-mpeg-dsmcc.c b/epan/dissectors/packet-mpeg-dsmcc.c
index 54e80c3..e5c0402 100644
--- a/epan/dissectors/packet-mpeg-dsmcc.c
+++ b/epan/dissectors/packet-mpeg-dsmcc.c
@@ -535,7 +535,7 @@ dissect_dsmcc_un_download(tvbuff_t *tvb, packet_info *pinfo, proto_tree *tree,
 	msg_id = tvb_get_ntohs(tvb, offset+2);
 
 	pi = proto_tree_add_text(tree, tvb, 0, -1, "User Network Message - %s",
-			val_to_str(msg_id, dsmcc_dd_message_id_vals, "%s"));
+			val_to_str(msg_id, dsmcc_dd_message_id_vals, "%u"));
 	sub_tree = proto_item_add_subtree(pi, ett_dsmcc_payload);
 
 	switch (msg_id) {
-- 
1.7.10.4

